config.json

```diff
@@ -109,13 +109,7 @@
         "name": "Elyses Analytic Enchantments",
         "uuid": "45d956db-d4ef-4468-b1d3-47021f172c15",
         "concepts": ["array-analysis"],
-        "prerequisites": [
-          "arrays",
-          "booleans",
-          "callbacks",
-          "arrow-functions",
-          "numbers"
-        ],
+        "prerequisites": ["arrays", "booleans", "callbacks", "numbers"],
```

Comment:
This exercise iteration no longer teaches arrow functions, and no other exercise does. Leaving it as a pre-requisite is a configlet error. What are you asking me to do here?

1. replace it as a pre-requisite? where should i specify it as being taught?
2. replace it as a taught concept even though it is not taught?
3. leave it removed, but change the example to not use arrow functions?

Reply:
Sorry, I didn't fully realize arrow functions are not included anymore. :see_no_evil: I would recommend to keep them in the introduction file and as a taught concept even though there are currently not practiced in the exercise. Then this can be merged and the student at least had a chance to read the introduction content. From that state, I can create an issue about teaching arrow functions in another exercise (or move it to another existing exercise) and once that is ready, it can be fully removed from this exercise. I will include the removal as a task in the issue. WDYT?

Reply:
Sounds fair to me, if that is easier to track for future work then can do that.

Re-added in https://github.com/exercism/javascript/pull/1725/commits/ff230366db5d185188a26c6db4e72f6c91834291